eox_nelp/signals/receivers.py

```diff
@@ -63,7 +64,7 @@ def create_course_notifications(course_key, **kwargs):  # pylint: disable=unused
     create_course_notifications_task.delay(course_id=str(course_key))
 
 
-def certificate_publisher(certificate, **kwargs):  # pylint: disable=unused-argument
+def certificate_publisher(certificate, metadata, **kwargs):  # pylint: disable=unused-argument
```

Comment:
```suggestion
def certificate_publisher(certificate, metadata, *args, **kwargs):  # pylint: disable=unused-argument
```

?

Reply:
IMO that is not necessary, since the signal doesn't' send any args